content/includes/common-and-popular.html

```diff
@@ -20,7 +20,7 @@ <h3 class="va-h-ruled">Popular on vets.gov</h3>
       <li>
         <a href="/gi-bill-comparison-tool/">View education benefits available by school</a>
       </li>
-      <li><a target="_blank" href="https://www.veteranscrisisline.net/" class="external">Contact the Veterans Crisis Line</a></li>
+      <li><a target="_blank" href="https://www.veteranscrisisline.net/" class="external"><span class="usa-sr-only"></span>Contact the Veterans Crisis Line</a></li>
```

Comment:
@markgreenburg, the following regex might facilitate the transition if we have determined the appropriate pattern:
Search: `<a (.*?href="http.*?)(.*?)>(.*?)</a>`
Replace (Content): `<a $1$2><span class="usa-sr-only"></span>$3</a>`
Replace (**/*.jsx): `<a $1$2><span className="usa-sr-only"></span>$3</a>`

Reply:
That's a good idea. From what I've seen, some of the content already has the correct spans. One caveat I can see is that the `md` content pages may not work with this.

Reply:
Thanks @markgreenburg, I will remove duplicates. Could you please clarify what you anticipate the problem being with the `md` content pages?

Reply:
I'm still not sure how our markdown-based content pages are generated in the build, but if they're written in markdown which is then compiled to html, I'm assuming we'd need to either change how links are transpiled from md to html, or add the spans to the html output directly.  I may have a fundamental misunderstanding of how these content pages work, though.

Reply:
Thanks for clarifying @markgreenburg, I'll investigate this.

Reply:
@markgreenburg, it looks like the best way to make this consistent is to convert the external markdown links into html first.

Reply:
Oh right, I forgot you can just inline html within markdown. That seems pretty reasonable to me.

Reply:
Sounds good @markgreenburg, thanks for the reminder about the markdown links.